envi/archs/arm/__init__.py

```diff
@@ -52,8 +50,6 @@ class ThumbModule(envi.ArchitectureModule):
     '''
     This architecture module will *not* shift to ARM mode.  Evar.
     '''
-    #Current ARM version working in - Not fully integrated yet
-    archVersion = ('ARMv7A')
```

Comment:
actually, i was mistaken.  please don't add to what's there for another string.  look at the __init__() function.  it accepts in the name and hands it in to the parent class __init__() as name.  don't store it again, please.  what you can store is the bitmask.  the name or bitmask should be handed into the disassemblers (_arch_dis and _arch_thumb_dis) and the bitmask should be stored there.  if you hand in the name, it should then be converted to the bitmask before being stored; although the name could be stored there as well.